### PR TITLE
update Analytics Zoo logo in readme.md

### PR DESCRIPTION
**Pull request by helenlly**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# Analytics Zoo
+<div align="center">
+  <img src="https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/img/logo.jpg"><br>
```
Comment (jason-dai) [inline README.md]: the image is too large

File: README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+</div>
+
+--------
```
Comment (jason-dai) [inline README.md]: remove this bar

File: README.md (hunk 1 above)
Comment (hkvision) [inline README.md]: make it smaller and centralize it

File: README.md [hunk 3]
```diff
@@ -1,4 +1,9 @@
...
+# 
 *A unified analytics + AI platform for __distributed TensorFlow, Keras and BigDL on Apache Spark__*
```
Comment (hkvision) [inline README.md]: Add `Analytics Zoo: `?

File: README.md (hunk 1 above)
Comment (helenlly) [reply README.md]: smaller now

File: README.md (hunk 2 above)
Comment (helenlly) [reply README.md]: removed

File: README.md (hunk 1 above)
Comment (helenlly) [reply README.md]: I see the image in center, and u?

Comment (hkvision) [reply README.md]: yeah

File: README.md (hunk 3 above)
Comment (helenlly) [reply README.md]: updated

File: README.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
-# Analytics Zoo
-*A unified analytics + AI platform for __distributed TensorFlow, Keras and BigDL on Apache Spark__*
+<div align="center">
+   <p align="center"> <img src="https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/img/logo.jpg" height=300px； weight=400px；"><br></p>
+</div>
+
+# 
+__Analytics Zoo：__*A unified analytics + AI platform for __distributed TensorFlow, Keras and BigDL on Apache Spark__*
```
Comment (hkvision) [inline README.md]: make this line larger? currently the headline is too small...

File: README.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
-# Analytics Zoo
-*A unified analytics + AI platform for __distributed TensorFlow, Keras and BigDL on Apache Spark__*
+<div align="center">
+   <p align="center"> <img src="https://github.com/analytics-zoo/analytics-zoo.github.io/blob/master/img/logo.jpg" height=300px； weight=400px；"><br></p>
```
Comment (hkvision) [inline README.md]: make the logo a little bit smaller?

**Pull request merged**